pyodide-build/pyodide_build/buildall.py

```diff
@@ -30,11 +31,13 @@ def __init__(self, returncode):
         super().__init__()
 
 
+@total_ordering
+@dataclasses.dataclass(eq=False, repr=False)
```

Comment:
It is not valid to set attributes directly on a class, that's very, very likely to not work as expected. These are actually data classes, which transforms the class attributes into instance attributes.